tagmanager/src/tm_source_file.h

```diff
@@ -14,14 +14,12 @@
 #include <glib.h>
 #include <glib-object.h>
 
+#include "tm_parser.h"
```

Comment:
it's probably not as nice, but as the type is a mere typedef anyway it could "just" go in some other installed file instead of *tm_parser.h* if we don't want to expose more stuff